.github/workflows/integrate.yaml

```diff
@@ -52,5 +52,8 @@ jobs:
         run: |
             pipenv run python -m pytest --cov=yuptoo tests
 
-
-
+      - name: Upload coverage to codecov
```

Comment:
@upadhyeammit / @patilsuraj767  - do you think we should create separate job for codecov OR is it ok to keep it here?

Reply:
I think it is better to have a separate job for codecov so that it will not need to wait for pytest to finish and will run in parallel with Lint and Test job. 
Also, if codecov runs the unit test should we consider to remove the old Test job?

Reply:
> Also, if codecov runs the unit test should we consider to remove the old Test job?

@patilsuraj767,  pytest provides coverage integration. In above step, it runs test using `--cov`flag. In my opinion we can keep it under Test job only.

There is alternative to pytest-cov. Refer -  https://coverage.readthedocs.io/en/7.0.1/
Thoughts?

Reply:
Hello @kgaikwad I have gone through the documentation and found that `codecov/codecov-action@v3` action just uploads the report and does not run the unit test. Hence we should keep this under the same test job.